.github/actions/deploy-service/action.yaml

```diff
@@ -11,6 +11,9 @@ inputs:
     workspace:
         description: "The name of the workspace to deploy the service into."
         required: true
+    custom_artefact_location:
```

Comment:
Maybe pedantic and/or something for next iteration but custom_artefact_location implies a custom artefact and it isn't ... it is a standard artefact for whichever domain repo but the location (sub directory in the S3 bucket) is default. Would a better name for this variable reflect that it is the name of the s3 sub-directory - eg alternative_artefact_location or default_artefact_location (since I think it will always be default)...

Reply:
Suggested name: artefact_sub_directory